src/app/product/createProduct/page.tsx

```diff
@@ -0,0 +1,9 @@
+import AddProductForm from "@/app/product/createProduct/addForm";
+
+export default function CreateProduct() {
+  return (
+    <>
+      <AddProductForm ></AddProductForm>
```

Comment:
if this component is only to call AddProductForm component, you can have that code here

Reply:
ok sure .updated code.